flask_jwt_extended/utils.py

```diff
@@ -153,10 +155,12 @@ def create_access_token(identity, fresh=False, expires_delta=None, user_claims=N
                           'JWT_ACCESS_TOKEN_EXPIRES` config value
                           (see :ref:`Configuration Options`)
     :param user_claims: Optional JSON serializable to override user claims.
+    :param headers: Optional
     :return: An encoded access token
     """
     jwt_manager = _get_jwt_manager()
-    return jwt_manager._create_access_token(identity, fresh, expires_delta, user_claims)
+    return jwt_manager._create_access_token(identity, fresh, expires_delta, user_claims,
+                                            headers=headers)
 
 
 def create_refresh_token(identity, expires_delta=None, user_claims=None):
```

Comment:
Can we add this option to the creation of refresh tokens as well?

Reply:
Yes, I was actually adding this in refresh token but IDK why I missed it anyway, Thanks for pointing this out. I will add this as well.